bit_manipulation/binary_or_operator.py

```diff
@@ -22,17 +22,17 @@
 
 def Binary_OR_Operator (a : int, b : int):
     
-    if type(a) == float or type(b) == float:
+    if isinstance(a, float) or isinstance(b, float):
         raise TypeError("'Float' object cannot be implemented as an integer")
-    if type(a) == str or type(b) == str:
+    if isinstance(a, str) or isinstance(b,str):
         raise TypeError("'str' object cannot be implemented as an integer")
-    if a <0 or b < 0:
+    if a < 0 or b < 0:
         raise ValueError("the value of both input must be positive")
     """
     a_binary and b_binary are the binary of a and b in str format
     """
-    a_binary = convert_to_binary(a)
-    b_binary = convert_to_binary(b)
+    a_binary = str(bin(a))[2:]
+    b_binary = str(bin(b))[2:]
     binary = []
     if len(a_binary) > len(b_binary):

```

Comment:
How could str.zfill() simplify the code?  https://docs.python.org/3/library/stdtypes.html#str.zfill